src/runtime/components/data/Table.vue

```diff
@@ -56,8 +56,8 @@
 
         <template v-else>
           <tr v-for="(row, index) in rows" :key="index" :class="[ui.tr.base, isSelected(row) && ui.tr.selected, $attrs.onSelect && ui.tr.active, row?.class]" @click="() => onSelect(row)">
-            <td v-if="modelValue" :class="ui.checkbox.padding">
-              <UCheckbox v-model="selected" :value="row" aria-label="Select row" @click.stop />
+            <td v-if="modelValue || $slots['row-select']" :class="ui.checkbox.padding">
```

Comment:
Where does this `row-select` slot comes from?

Reply:
My bad, I was experimenting w/ slots for a more flexible approach but it introduced breaking changes so I refactored. I can remove this...